home/templates/grades.html

```diff
@@ -195,10 +195,16 @@ <h2>Lookup by professor</h2>
 
 	function generate_graph(grade_data, course, semester=null, section=null) {
 		course = course.toUpperCase();
-		$("#grades-graphs").append('<p id="grades-text' + canvasCount + '" class="text-center grades-info"></p>')
-		$("#grades-graphs").append('<div class="graph-wrapper"><canvas id="grades' + canvasCount + '" width="500px" height="500px" style="margin: 0 auto;"></canvas></div>')
-
-		var graph = document.getElementById('grades' + canvasCount).getContext("2d");
+		// Create a new row for every third graph
+		if (canvasCount % 2 == 0) {
+			$("#grades-graphs").append('<div class="row"></div>');
+		}
+		// Create a new column to hold the graph and append to the rows
+		let $graphColumn = $('<div class="col"></div>');
+		$("#grades-graphs .row:last-child").append($graphColumn);
+		// Add the canvas and text elements to the column
+		$graphColumn.append('<p id="grades-text' + canvasCount + '" class="text-center grades-info"></p>');
```

Comment:
I assume the $ prefix here denotes a jquery object? This seems like syntax that could potentially be convenient, but let's avoid using it for now since it isn't used anywhere else in the codebase.